components/style/themes/default.less

```diff
@@ -383,6 +383,11 @@
 @menu-item-active-bg: @item-active-bg;
 @menu-item-active-border-width: 3px;
 @menu-item-group-title-color: @text-color-secondary;
+@menu-item-vertical-margin: 4px;
+@menu-item-font-size: @font-size-base;
+@menu-item-last-margin: 8px;
+@menu-icon-size: @font-size-base;
+@menu-icon-size-lg: @font-size-lg;
```

Comment:
`menu-icon-size` may not good. Could you remove icon related in this PR?

Reply:
hi! Any suggests? Why not good)

Reply:
We hope icon size is follow the current font-size instead of specific variable cause user have to do additional config for this. icon size should not provides if possible.

Reply:
@alxkosov set default values for this

Reply:
@zombieJ , styles for `menu-icon-size` was deleted